build.psm1

```diff
@@ -2377,6 +2377,7 @@ function Start-CrossGen {
             "Microsoft.PowerShell.CoreCLR.Eventing.dll",
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Validate simple Get-LocalGroupMember.Can get a local group member by name
```powershell
Expected 'Local', but got Unknown.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.LocalAccounts\Pester.Command.Cmdlets.LocalAccounts.LocalGroupMember.Tests.ps1: line 269
269:                 $result.PrincipalSource | Should -Be Local
```